e2e/Makefile

```
@@ -83,6 +83,7 @@ functional-test: install-godog
 	ELASTIC_APM_ENVIRONMENT="${ELASTIC_APM_ENVIRONMENT}" \
 	ELASTIC_APM_SECRET_TOKEN="${APM_SECRET_TOKEN}" \
 	ELASTIC_APM_SERVER_URL="${APM_SERVER_URL}" \
+	KIBANA_VERSION=pr96718 \
```

Let's use the backported-to-7.x PR for the 96718 PR.
```suggestion
	KIBANA_VERSION=pr97021 \
```